tests/Scripts/Configure-AzureSearchFunctionalTests.ps1

```diff
@@ -0,0 +1,8 @@
+[CmdletBinding(DefaultParameterSetName='RegularBuild')]
+param (
+    [string]$Slot
+)
+
+Write-Host "Setting environment variable for the slot: " $Slot
+$env:Slot = $Slot
+Write-Host "##vso[task.setvariable variable=Slot;]$Slot"
```

Comment:
I am thinking of removing this config script all together and adding the "Slot" flag to the test configuration json directly through "Import-Configuration.ps1", similar to that of the Gallery functional tests... thoughts here?

Reply:
Yeah, I think we should our patterns. Gallery's method seems good enough.